editor/sidebar/post-status/index.js

```diff
@@ -40,15 +47,17 @@ class PostStatus extends Component {
 
 		return (
 			<PanelBody title={ __( 'Status & Visibility' ) }>
-				<div className="editor-post-status__row">
-					<label htmlFor={ pendingId }>{ __( 'Pending review' ) }</label>
-					<FormToggle
-						id={ pendingId }
-						checked={ status === 'pending' }
-						onChange={ onToggle }
-						showHint={ false }
-					/>
-				</div>
+				{ ! isPublished &&
```

Comment:
What would it take to be able to show this for published posts as well? I'd suppose we need to revert back to the post's original status when toggling off, not a static value. Maybe an action that clears the current edited value for a property (we might want that for what we have anyways).

I'm not sure it makes much sense to revert from Published to Pending Review, but it is possible in the current editor.

Maybe if we at least have flow for reverting to draft it becomes a little more reasonable, since there's a workflow for changing from Published to Pending Review.